src/pages/SwapPage/layout/Checkout/SwapForm.tsx

```diff
@@ -0,0 +1,209 @@
+	approved: boolean;
+	isLoading: boolean;
+}
+
+export const SwapForm: React.FC<IProps> = ({
+	tokens,
+	ApproveSwapTokens,
+	SwapTokens,
+	handleSetFromToken,
+	handleSetToToken,
+	handleSetAmountIn,
+	handleSetMinAmountOut,
+	fromSupertoken,
+	toSupertoken,
+	amountIn,
+	minAmountOut,
+	approved,
+	isLoading,
+}) => {
+	return (
+		<React.Fragment>
+			<Grid
+				container
+				spacing={3}
+				style={{
+					backgroundColor: '#2d3233',
+				}}
+			>
+				<Grid item xs={12} sm={12}>
+					<Typography
+						variant="h6"
+						gutterBottom
+						style={{
+							color: 'white',
+							fontSize: 'larger',
+						}}
+					>
+						From Supertoken
+					</Typography>
+					<select
+						name="SuperTokens"
+						id="supertoken"
+						onChange={async (e) => await handleSetFromToken(e.target.value)}
+						className={styles.select}
+						style={{
+							color: 'lightgray',
+							backgroundColor: '#2b2b2b',
+							border: 'none',
+							fontSize: 'large',
+							paddingLeft: '1em',
+						}}
+					>
+						<option value={''}>Choose a Token</option>
+						{tokens.map((token, i) => {
+							return (
+								<option key={`${token.name}-${i}`} value={token.address}>
+									{token.name}
+								</option>
+							);
+						})}
+					</select>
+				</Grid>
+
+				<Grid item xs={12} style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+					<SwapHorizIcon
+						style={{
+							color: 'white',
+						}}
+					/>
+				</Grid>
+
+				<Grid item xs={12} sm={12}>
+					<Typography
+						variant="h6"
+						gutterBottom
+						style={{
+							color: 'white',
+							fontSize: 'larger',
+						}}
+					>
+						To Supertoken
+					</Typography>
+					<select
+						name="SuperTokens"
+						id="supertoken"
+						onChange={async (e) => await handleSetToToken(e.target.value)}
+						className={styles.select}
+						style={{
+							color: 'lightgray',
+							backgroundColor: '#2b2b2b',
+							border: 'none',
+							fontSize: 'large',
+							paddingLeft: '1em',
+						}}
+					>
+						<option value={''}>Choose a Token</option>
+						{tokens.map((token, i) => {
+							return (
+								<option key={`${token.name}-${i}`} value={token.address}>
+									{token.name}
+								</option>
+							);
+						})}
+					</select>
+				</Grid>
+
+				<Grid item xs={12}>
+					<label
+						style={{
+							color: 'white',
+							marginBottom: '20px',
+						}}
+					>
+						Input Amount
+					</label>
+					<input
+						required
+						value={amountIn}
+						onChange={async (e) => await handleSetAmountIn(e.target.value)}
+						style={{
+							color: 'white',
+							backgroundColor: '#2b2b2b',
+							width: '100%',
+							height: '6vh',
+							paddingLeft: '1em',
+							border: 'none',
+							fontSize: 'large',
+						}}
+					/>
+				</Grid>
+				<Grid item xs={12}>
+					<label
+						style={{
+							color: 'white',
+							marginBottom: '20px',
+						}}
+					>
+						Minimum Output Amount
```

Comment:
Calculate this for the use like is done on every other swap UI, take the current price for the token and compute a minimum output amount using the current price - 1% for slippage
<img width="1610" alt="Screen Shot 2022-09-17 at 11 16 08 AM" src="https://user-images.githubusercontent.com/5077880/190864065-ae7e8dfa-66c2-498d-8dd7-b9277983d4dc.png">

Reply:
@Seroxdesign, you missed change request